src/lib/shapes/data/polygonclippath.ts

```diff
@@ -0,0 +1,9 @@
+export const clipPathMap = {

```

Comment:
Can you please rename this file to `polygonClipPath.ts`?

Reply:
Sure.